aws/cloudformation/drone-stack.yml

```diff
@@ -269,6 +269,7 @@ Resources:
             -
               Name: DRONE_S3_BUCKET
               Value: code-dot-org
+              # TODO: change this to: !Ref CiCacheBucket
```

Comment:
It might be necessary to change this, but if we do it'll recreate the drone master ecs task. That's one too many variables for me at the moment, so I'm just leaving a TODO for now.